Makefile

```diff
@@ -39,7 +39,6 @@ coverage:
 	open htmlcov/index.html
 
 build-docs:
-	pip install -e .[doc]
 	cd docs/; sphinx-build -W -T -E . _build/html
```

Comment:
I guess the idea here was that one would be able to run `make build-docs` without the assumption that docs dependencies were already installed.

Reply:
Yeah, if it was really fast, I probably wouldn't object, but the pip install actually takes a while. So if you run `make build-docs` over and over, it gets old really fast. Also, a bunch of our other `make` targets assume that you have the appropriate installed tools (wheel, setuptools, towncrier, etc). We could probably document them all better, though. :+1: